### PR TITLE
Auto release Docker images

### PR DESCRIPTION
**Pull request by elegaanz**
I tried something, not sure it will work… My biggest concern is that I pull the latest version of the Docker image to tag it, but what if it didn't finished building yet?

If you could give this a look @trinity-1686a it would be great, you are the CI expert here :sweat_smile: (but if you don't want/can't it's fine too, don't worry)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -228,6 +229,18 @@ jobs:
...
+    executor:
+      name: default
+    steps:
+    - restore_env:
+        cache: none
+    - run: |
+        docker login -p ${DOCKER_PASSWORD} -u ${DOCKER_LOGIN}
+        docker pull plumeorg/plume:latest
+        docker tag plumeorg/plume:latest plumeorg/plume:v<< pipeline.git.tag >>
+        docker push plumeorg/plume:v<< pipeline.git.tag >>
```
Comment (igalic) [inline .circleci/config.yml:242]: note the inconsistency here

File: .circleci/config.yml [hunk 2]
```diff
@@ -255,3 +268,7 @@ workflows:
           branches:
             only:
               - /^master/
+    - docker release:
+        filters:
+          tags:
+            only: /.*/
```
Comment (igalic) [inline .circleci/config.yml:274]: vs here, where we only use numbers (which is why this line: https://github.com/Plume-org/Plume/pull/790/files#diff-7f1fef10867a0b0e04b203d80a095852R8 exists)

File: .circleci/config.yml (hunk 1 above)
Comment (elegaanz) [reply .circleci/config.yml:242]: It is how we started naming them on Docker : https://hub.docker.com/repository/docker/plumeorg/plume/general

But maybe we should do that everywhere, yes?

File: .circleci/config.yml (hunk 2 above)
Comment (trinity-1686a) [reply .circleci/config.yml:274]: I don't know how circle-ci handle absent tag, if this is a regex (I think it is? Would have to read the docs to make sure, `/.+/ ` would be better as it require tag to be at list one character

File: .circleci/config.yml [hunk 3]
```diff
@@ -228,6 +229,18 @@ jobs:
...
     - run: crowdin upload -b master
+  docker release:
+    executor:
+      name: default
+    steps:
+    - restore_env:
+        cache: none
+    - run: |
+        docker login -p ${DOCKER_PASSWORD} -u ${DOCKER_LOGIN}
+        docker pull plumeorg/plume:latest
```
Comment (trinity-1686a) [inline .circleci/config.yml:240]: I'm pretty sure this won't work. Best case this job is ran last, and auto-build at docker is fast enough, it works. Worst (and more likely), this job is not ran last/docker is slower/whatever, and the wrong image get tagged.

File: .circleci/config.yml (hunk 1 above)
Comment (igalic) [reply .circleci/config.yml:242]: i'll drop the option in the other PR.

**Comment by elegaanz** [comment]
I made some research, and found out that this is configurable from the Docker Hub UI. I added a trigger that should create a new tag on Docker Hub for each new tag on Github.

**Pull request closed without merging**